docs/docs/reference/built-in-components/gatsby-head.md

```diff
@@ -57,6 +57,28 @@ export default Page
 export { Head } from "../another/location"
 ```
 
+### Editing `<html>` and `<body>`
+
+> Support for the editing `<html>` and `<body>` added in `gatsby@5.5.0`.
```

Comment:
Optimistic version - might need to update if feature doesn't land in next minor release